api-extractor/src/definitions/ApiDocumentation.ts

```diff
@@ -412,28 +396,6 @@ export default class ApiDocumentation {
         this.reportError(`Unexpected token: ${token.type} ${token.tag} ${token.text}`);
       }
     }
-
-    if (apiTagCount > 1) {
-      this.reportError('More than one API Tag was specified');
-    }
-
-    if (this.apiItem instanceof ApiPackage) {
-      if (apiTagCount > 0) {
-        const tag: string = '@' + ApiTag[this.apiTag].toLowerCase();
-        this.reportError(`The ${tag} tag is not allowed on the package, which is always public`);
-      }
-      this.apiTag = ApiTag.Public;
-    }
-
-    if (this.preapproved) {
-      if (this.apiTag !== ApiTag.Internal) {
-        this.reportError('The @preapproved tag may only be applied to @internal defintions');
-        this.preapproved = false;
-      } else if (!(this.apiItem.getDeclarationSymbol().flags & (ts.SymbolFlags.Interface | ts.SymbolFlags.Class))) {
-        this.reportError('The @preapproved tag may only be applied to classes and interfaces');
```

Comment:
This test needs to be done from ApiItem.  The other tests here are purely syntax checks for the JSDoc string, so they can stay in ApiDocumentation.